### DISCUSSION TITLE
My first contribution.

### DISCUSSION THREAD

**Discussion by onesch** [space-wizards/space-station-14#32937] [Questions]
I recently started getting interested in the ss14 code, want to ask those who have already contributed, I want to add new food and their recipes, is this welcome?

**Comment by lzk228** [comment]
food is kinda in freeze, based on #8524 
but it's not like "you can't add food" but "you can add food but it can be deleted on cooking rework"
and my last food pr got merged #30668

**Reply by onesch** [reply]
> food is kinda in freeze, based on #8524 
> but it's not like "you can't add food" but "you can add food but it can be deleted on cooking rework"
> and my last food pr got merged #30668 

you made food for moths, submitted a pull request and it was approved?

**Reply by lzk228** [reply]
yeah, feel free to make your own pr, then maintainers will take it for a discussion in discord and either will request some changes either close it while submitting reasons for it

**Reply by onesch** [reply]
> yeah, feel free to make your own pr, then maintainers will take it for a discussion in discord and either will request some changes either close it while submitting reasons for it

How long have you been contributing to ss14?

**Reply by lzk228** [reply]
like half and a year, but i think in case of small prs it's not really matter who made the pr, a newbie or an old contrib, if you fear that your pr won't be accepted

**Reply by onesch** [reply]
> like half and a year, but i think in case of small prs it's not really matter who made the pr, a newbie or an old contrib, if you fear that your pr won't be accepted

ok, i learned something new, thanks.

**Reply by lzk228** [reply]
glad to help

**Comment by crossterra-agent** [comment]
- **Status**: DONE
- **Role**: orchestrator
- **Link**: https://github.com/CrossTerra/monorepo/pull/2037
- **Summary**: PR ready for human review — all 3 plan steps implemented with tests, CI green (duplicate developer status updates received; no new action needed).
